pages/verify.tsx

```diff
@@ -102,49 +112,33 @@ export default function VerifyPage() {
           </div>
         )}
         {!signup && !verified && (expired || obsolete) && (
-          <div className="centeredElement column">
-            <h1 className="centeredElement">
+          <div>
+            <h1 className="text-3xl font-bold text-center">
               Link is {expired ? "expired" : "obsolete"}!
             </h1>
-            <div>No worries, we can send you a new one </div>
+            <div className="mt-8">No worries, we can send you a new one </div>
           </div>
         )}
         {!verified && !signup && !expired && !obsolete && !email && (
           <div>
-            <h1>loading ...</h1>
-            <CircularProgress />
+            <h1 className="text-3xl font-bold text-center">loading ...</h1>
+            <Spinner />
           </div>
         )}
         {!signup &&
           !verified &&
           (!!email || !!expired || !!obsolete) &&
           !disabled && (
-            <Button
-              variant="contained"
-              type="button"
-              className="marginTopMedium"
-              onClick={() => resendLink()}
-            >
+            <Button type="button" onClick={() => resendLink()}>
```

Comment:
This should be possible too and apply to all files:
```suggestion
            <Button type="button" onClick={resendLink}>
```

Reply:
will do!